x/ccv/provider/keeper/keeper.go

```diff
@@ -761,12 +759,19 @@ func (k Keeper) GetSlashAcks(ctx sdk.Context, chainID string) []string {
 }
 
 // ConsumeSlashAcks empties and returns the slash acks for a given chain ID
-func (k Keeper) ConsumeSlashAcks(ctx sdk.Context, chainID string) (acks []string) {
-	acks = k.GetSlashAcks(ctx, chainID)
-	if len(acks) < 1 {
+func (k Keeper) ConsumeSlashAcks(ctx sdk.Context, chainID string) (acks []types.ConsumerConsAddress) {
+	addresses := k.GetSlashAcks(ctx, chainID)
+	if len(addresses) < 1 {
 		return
 	}
+
+	acks, err := StrToConsumerConsAddresses(addresses)
+	if err != nil {
+		// todo
```

Comment:
shall return err ? if yes, where to deal with the err ? @smarshall-spitzbart

Reply:
I'd say pattern match other places in the repo that panic or return upon error. See above on how consumer does this